core-primitives/api-client-extensions/src/chain.rs

```diff
@@ -13,6 +13,7 @@ pub type StorageProof = Vec<Vec<u8>>;
 pub trait ChainApi {
 	fn last_finalized_block(&self) -> ApiResult<Option<SignedBlock>>;
 	fn signed_block(&self, hash: Option<H256>) -> ApiResult<Option<SignedBlock>>;
+	fn get_blocks(&self, from: u32, to: u32) -> ApiResult<Vec<SignedBlock>>;
```

Comment:
I know this hasn't been done in this trait here - but could you add some documentation to this new trait method? I'd be especially interested in the range, if `to` is included or not.

Reply:
Yes, from and to is chosen such that no type conversion is necessary

Reply:
ah okay, good to know 👍